src/fairness/reader/data_reader_db.cpp

```diff
@@ -343,7 +364,9 @@ std::shared_ptr<weighted_tree_node_t> load_accounting_db (
     // open flux-accounting DB in read-write mode
     rc = sqlite3_open_v2 (path.c_str (), &DB, SQLITE_OPEN_READWRITE, NULL);
     if (rc != SQLITE_OK) {
-        std::cerr << "error opening DB: " << sqlite3_errmsg (DB) << std::endl;
+        std::stringstream msg;
+        msg << "error opening DB: " << sqlite3_errmsg (DB);
+        m_err_msg += msg.str ();
```

Comment:
Per @dongahn 's verbal comment, the `+=` is useful for recursive functions, but for single-invocation functions, using `=` is preferred to ensure stale errors are not included.